garak/resources/common.py

```diff
@@ -42,12 +43,21 @@
 
 
 def load_advbench(size: int = 0) -> pd.DataFrame:
-    advbench_path = (
+    advbench_base_path = (
         garak._config.transient.basedir
         / "resources"
         / "advbench"
         / "harmful_behaviors.csv"
     )
+    advbench_path = (
+        garak._config.transient.cache_dir
+        / "resources"
+        / "advbench"
+        / "harmful_behaviors.csv"
+    )
+    if advbench_base_path.is_file() and not advbench_path.is_file():
+        shutil.copy2(advbench_base_path, advbench_path)
+
```

Comment:
In a future iteration helper methods from `_config`  for loading existing resource files may be appropriate, allowing the user to place files in the cache_dir or data_dir to override shipped versions.